frontend/src/scenes/feature-flags/FeatureFlagSnippets.tsx

```diff
@@ -155,16 +168,33 @@ if (${conditional}) {
 export function GolangSnippet({
     flagKey,
     groupType,
+    payload,
+    encryptedPayload,
     multivariant,
     localEvaluation,
     samplePropertyName,
 }: FeatureFlagSnippet): JSX.Element {
     const clientSuffix = 'client.'
```

Comment:
lol you didn't do this but what a terrible variable name – it's a prefix, not a suffix!